### DISCUSSION TITLE
最新版本的bot，进行uid查询，部分会出现空白页面bug

### DISCUSSION THREAD

**Discussion by YYLMZXC** [Arondight/Adachi-BOT#518] [Q&A] [answered]
![ZGHQ34V0)OPWPPIW E`9X9G](https://user-images.githubusercontent.com/90677130/147095484-74285d9a-b9cb-4031-af0d-f3db50123826.png)
![D3C _VKB7VCD 9AZT5W{RL4](https://user-images.githubusercontent.com/90677130/147095492-328afbea-74dd-479e-8987-b51bc6b478d3.png)
![M@1T)@FN0GP01PIK`O _QSU](https://user-images.githubusercontent.com/90677130/147095499-5463c7e5-f38c-4028-b1fe-b750114c851c.png)
![D3C _VKB7VCD 9AZT5W{RL4](https://user-images.githubusercontent.com/90677130/147096072-c2565dc4-286d-48b8-85fd-565b3dd5fc44.png)
![ZGHQ34V0)OPWPPIW E`9X9G](https://user-images.githubusercontent.com/90677130/147096167-c3a613c2-9e79-48c5-b2db-5681367421fc.png)

目前推测原因应该是查询无元素主角，没有找到元素导致的bug

**Comment by mark9804** [comment]
好像不是无元素主角的问题，我再看看，已经同步到issue #519 了

**Comment by mark9804** [comment, marked as answer]
这个问题已经修复，麻烦拉取一下最新的代码看看

**Reply by YYLMZXC** [reply]
ok，已解决，谢谢
![image](https://user-images.githubusercontent.com/90677130/147110026-47afae42-cd91-47a8-855e-c0cebf8dda45.png)
